AbandonedCartReminder.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/*************************************************************************************/
+/*      This file is part of the Thelia package.                                     */
+/*                                                                                   */
+/*      Copyright (c) OpenStudio                                                     */
+/*      email : dev@thelia.net                                                       */
+/*      web : http://www.thelia.net                                                  */
+/*                                                                                   */
+/*      For the full copyright and license information, please view the LICENSE.txt  */
+/*      file that was distributed with this source code.                             */
+/*************************************************************************************/
+
+namespace AbandonedCartReminder;
+
+use Propel\Runtime\Connection\ConnectionInterface;
+use Thelia\Core\Translation\Translator;
+use Thelia\Install\Database;
+use Thelia\Model\Base\MessageQuery;
+use Thelia\Model\Lang;
+use Thelia\Model\LangQuery;
+use Thelia\Model\Message;
+use Thelia\Module\BaseModule;
+
+class AbandonedCartReminder extends BaseModule
+{
+    /** @var string */
+    const DOMAIN_NAME = 'abandonedcartreminder';
+
+    const VAR_DELAI_RAPPEL_1 = 'delai_rappel_1_en_minutes';
+    const VAR_DELAI_RAPPEL_2 = 'delai_rappel_2_en_minutes';
+    const VAR_CODE_PROMO_RAPPEL_2 = 'code_promo_rappel_2';
+
+    const MESSAGE_RAPPEL_1 = 'panier-abandonnes-message-rappel-1';
+    const MESSAGE_RAPPEL_2 = 'panier-abandonnes-message-rappel-2';
+
+    public function postActivation(ConnectionInterface $con = null)
+    {
+        $database = new Database($con);
+        $database->insertSql(null, [__DIR__ . '/Config/thelia.sql']);
```

Comment:
With this at each activation you will erase all data already registred

Reply:
Should I add something to keep old data ?

Reply:
Something like that ?
```php
        try {
            AbandonedCartQuery::create()->findOne();
        } catch (\Exception $e) {
            $database = new Database($con);
            $database->insertSql(null, [__DIR__ . "/Config/thelia.sql"]);
        }
```